plugins/amd-gpu/fu-amd-gpu-device.c

```diff
@@ -94,8 +94,6 @@ fu_amd_gpu_device_probe(FuDevice *device, GError **error)
 		fu_device_set_logical_id(device, "rom");
 		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE);
 		fu_udev_device_add_open_flag(FU_UDEV_DEVICE(device), FU_IO_CHANNEL_OPEN_FLAG_READ);
-		fu_udev_device_add_flag(FU_UDEV_DEVICE(device),
-					FU_UDEV_DEVICE_FLAG_VENDOR_FROM_PARENT);
```

Comment:
@superm1 please can you check this, I have no AMD GPU hardware. Thanks!

Reply:
Here is before change:  [get-devices.txt](https://github.com/user-attachments/files/16800337/get-devices.txt)
Here is after change: [get-devices.txt](https://github.com/user-attachments/files/16800324/get-devices.txt)

Reply:
those are both with `fwupdtool get-devices --plugins amd-gpu -vv`

Reply:
That looks identical to me -- thanks.